io/example/mets.xml

```diff
@@ -1,126 +1,350 @@
 <?xml version="1.0" encoding="UTF-8" ?>
+    <!-- 
+         Example METS file for the OCR-D Workflow.
+         http://ocr-d.de/
+         This file contains (or at least a reference to) all collected information of a
```

Comment:
Vorschlag: This container file holds (references to) all information and resources that constitute the digital object.

Reply:
OK